src/lib/keys.js

```diff
@@ -222,3 +222,30 @@ export const segmentNetworkKey = hex => {
 
   return [rowFrom(2), rowFrom(18), rowFrom(34), rowFrom(50)];
 };
+
+// ctsy @yosoyubik
+/**
+ * @param {object} pair
+ * @param {number} point
+ * @param {number} revision
+ * @return {string}
+ */
+export const compileMultikey = (point, pair1, pair2, revision1, revision2) => {
+  const bnsec1 = new BN(createRing(pair1), 'hex');
+  const bnsec2 = new BN(createRing(pair2), 'hex');
+
+  const tag = noun.dwim(noun.Atom.fromString('1'), noun.Atom.yes);
```

Comment:
Looking at this now, I think it should be `noun.Atom.fromInt(1)` so:

`const tag = noun.dwim(noun.Atom.fromInt(1), noun.Atom.yes);`